pkg/agent/apiserver/handlers/networkpolicy/handler.go

```diff
@@ -77,9 +68,13 @@ func newFilterFromURLQuery(query url.Values) (*querier.NetworkPolicyQueryFilter,
 		}
 	}
 	strSourceType := strings.ToUpper(query.Get("type"))
-	npSourceType, ok := mapToNetworkPolicyType[strSourceType]
-	if strSourceType != "" && !ok {
-		return nil, "", fmt.Errorf("invalid policy source type. Valid values are K8sNP, ACNP, ANNP and ANP (deprecated)")
+	var policyType cpv1beta.NetworkPolicyType
+	if strSourceType != "" {
+		npSourceType, ok := querier.NetworkPolicyTypeMap[strSourceType]
+		if !ok {
+			return nil, "", fmt.Errorf("unknown policy type. Valid types are K8sNP, ACNP, ANNP, BANP or ANP")
```

Comment:
Need to update doc too: https://github.com/antrea-io/antrea/blob/7d8d4b1eb31b3747f625634985f197bc92accac2/docs/antctl.md?plain=1#L292

Reply:
Thanks for the reminder. Done